content/en/network_monitoring/devices/guide/device_profiles.md

```diff
@@ -102,44 +171,165 @@ Adding tags to tabular metrics is similar to adding global tags, with two additi
+Once a profile is applied, you cannot bring it back to draft status.
+
+  {{< img src="/network_device_monitoring/profile_onboarding/device_status.png" alt="Screenshot of a device profile showing the draft state status" style="width:50%;">}}
+
+- **Filters**: The filters include the following options:
+  - Custom Profiles - User created device profiles.
+  - Created by Datadog - Datadog out-of-the-box profiles that can be viewed and used to build your own custom profile.
+  - Draft Profiles - Devices profiles in that are in draft mode.
 
-{{< img src="/network_device_monitoring/profile_onboarding/download_all_profiles_2.png" alt="The Network Device profile main page highlighting the Download All Profiles option" style="width:100%;">}}
+  {{< img src="/network_device_monitoring/profile_onboarding/device_filters.png" alt="Screenshot of the device profile inventory page showing the filter options" style="width:60%;">}}
+
+- **Create new profile and Download**: The **+ Create New Profile** button opens the profile creation form which allows you to [build a new device profile.](#build-device-profiles) Clicking the download button generates and download a `.zip` bundle which contains the `yaml` files for the profiles you created. <br></br>
+
+  {{< img src="/network_device_monitoring/profile_onboarding/create_profile_download.png" alt="Screenshot of the device profile inventory page showing the download and create a new profile buttons" style="width:50%;">}}
+
+- **Kebab menu**: Clicking the kebab menu to the right of a profile allows you to edit, clone, or delete the profile (for custom profiles only). You can also navigate to the **View related devices** on the NDM page, filtered to the device(s) the profile is applied to.<br></br>
+
+  {{< img src="/network_device_monitoring/profile_onboarding/device_kebab_menu.png" alt="Screenshot of the device profile inventory page showing the kebab menu on the right hand side" style="width:40%;">}}
+                                                  
+## Troubleshooting
+
+### What is a profile?
+* A profile is a configuration file that defines the metadata, metrics, and tags to collect from a device. See [metadata definition][17] for more information.
+
+### What is a device scan?
+* A device scan conducts a complete SNMP walk of the device, collecting all available data and forwarding it to Datadog for display in the UI. This process helps you identify the OIDs available on the device and add them to the profile for monitoring.
+
+### Why would I see no matching devices? 
+If no matching devices are found, it may be due to the following reasons:  
+  * **The profile is in Draft mode**:
+    * Draft profiles are not applied to the Agent. To start monitoring devices with your profile you must sync it to the Agent(s). This can be done by opening the profile and clicking on the [**Save & Sync Agents**](#apply-a-profile-to-created-devices) button.  
```

Comment:
```suggestion
    * Draft profiles are not applied to the Agent. To start monitoring devices with your profile, you must sync it to the Agent(s). This can be done by opening the profile and clicking on the [**Save & Sync Agents**](#apply-a-profile-to-created-devices) button.  
```